### ISSUE TITLE
Treatment of DMAC incorrect

### ISSUE THREAD

**Issue by graeme-winter** [winter-special-projects/python-cmsis-svd#1]
Illustrated by

```python
    "CHCTRLA": 0x0 | uctypes.UINT32,
    "CHCTRLB": 0x4 | uctypes.UINT8,
    "CHPRILVL": 0x5 | uctypes.UINT8,
    "CHEVCTRL": 0x6 | uctypes.UINT8,
    "CHINTENCLR": 0xC | uctypes.UINT8,
    "CHINTENSET": 0xD | uctypes.UINT8,
    "CHINTFLAG": 0xE | uctypes.UINT8,
    "CHSTATUS": 0xF | uctypes.UINT8,
```

at the end of  `DMAC.py` - multiple registers with same offset 🤔

**Comment by graeme-winter** [comment]
OK, these are in a cluster _inside_ the definition (unlike `PORT` which is itself a cluster)

Will need to define a `struct` of `struct`s here

**Comment by graeme-winter** [comment]
Fixed in 12d26153a98a0685678602b006073e15987b7c18
